internal/ansible/runner/runner.go

```diff
@@ -337,7 +337,7 @@ func (r *runner) makeParameters(u *unstructured.Unstructured) map[string]interfa
 		parameters = paramconv.MapToSnake(spec)
 	} else {
 		for k, v := range spec {
-			parameters[k] = v
+			parameters[k] = markUnsafe(v)
```

Comment:
We are also specifying the spec key as it is for json to Marshall (https://github.com/operator-framework/operator-sdk/pull/4566/files#diff-40a9a64aa847bb7816a7eb40cf6ec6b3ca70a7d4066e320d95fe9c0e0cb2ac11R349). Do we need to tag only these variables as unsafe?

Reply:
yeah we'll need to run this on all pieces of the spec that we pass in.